.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "libglnx"]
 	path = libglnx
 	url = https://git.gnome.org/browse/libglnx
-[submodule "libhif"]
+[submodule "libdnf"]
 	path = libdnf
```

Comment:
Not quite sure why the path was libdnf in the first place, if the submodule is libhif? If it fixes Jenkins it's probably ok...?